src/test/migrations/steps/template.ts

```diff
@@ -1,5 +1,5 @@
-import { BaseProtocol } from '@server/db'
-
-export default async (client: BaseProtocol) => {
-  // DB Step
-}
+// import { BaseProtocol } from '@server/db'
+//
```

Comment:
this file doesnt need to be changed

Reply:
Pre-commit hooks were failing because of "never used". That's why I commented out everything.

Reply:
In merge commits you can use --no-verify
👍